upup/pkg/fi/nodeup/nodetasks/bindmount.go

```diff
@@ -0,0 +1,260 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodetasks
+
+import (
+	"fmt"
```

Comment:
Should we start formatting our imports?  We have such differences across the project :(  I know nit pick :P

Reply:
My votes is for `goimports` style.  Not sure what k/k uses.

Reply:
Will fix the new files in this PR, but discuss that in another PR (I think we have discussion of it already in another issue - the shortcoming is that goimports doesn't remove blocks that have been added, IIRC.  If there's a better tool available that would be great...)